inst/qml/GeneralizedLinearModel.qml

```diff
@@ -0,0 +1,469 @@
+                CheckBox {
+                    name: "prsResVsYPlot"; label: qsTr("Residuals vs. fitted")
+                }
+                CheckBox {
+                    name: "prsResVsXPlot"; label: qsTr("Residuals vs. predictor")
+                }
+                CheckBox {
+                    name: "prsResQqPlot"; label: qsTr("Q-Q plot")
+                }
+
+            }
+
+            Group
+            {
+                title: qsTr("Quantile Residuals")
+                CheckBox {
+                    name: "quanResVsYPlot"; label: qsTr("Residuals vs. fitted")
+                }
+                CheckBox {
+                    name: "quanResVsXPlot"; label: qsTr("Residuals vs. predictor")
+                }
+                CheckBox {
+                    name: "quanResQqPlot"; label: qsTr("Q-Q plot")
+                }
+            }
+
+            Group
+            {
+                title: qsTr("Other Plots")
+                CheckBox {
+                    name: "partialPlot"; label: qsTr("Partial residual plots")
+                }
+                CheckBox {
+                    name: "zVsEtaPlot"; label: qsTr("Working responses vs. linear predictor")
+                }
+            }
+        }
+
+        Group {
+            title: qsTr("Show Outliers")
+            CheckBox {
+                name: "outlierQuanTable"; label: qsTr("Standardized quantile residuals: top")
+                childrenOnSameRow: true
+                IntegerField { name: "outlierQuanTableTopN"; defaultValue: 3}
+            }
+            CheckBox {
+                name: "outlierStdTable"; label: qsTr("Standardized deviance residuals: top")
+                childrenOnSameRow: true
+                IntegerField { name: "outlierStdTableTopN"; defaultValue: 3}
+            }
+            CheckBox {
+                name: "outlierStuTable"; label: qsTr("Studentized deviance residuals: top")
+                childrenOnSameRow: true
+                IntegerField { name: "outlierStuTableTopN"; defaultValue: 3}
+            }
+        }
+
+        Group {
+            title: qsTr("Show Influential Cases")
+            CheckBox {
+                name: "dfbetas"; label: qsTr("DFBETAS")
+            }
+            CheckBox {
+                name: "dffits"; label: qsTr("DFFITS")
+            }
+            CheckBox {
+                name: "covRatio"; label: qsTr("Covariance ratio")
+            }
+            CheckBox {
+                name: "cooksD"; label: qsTr("Cook's distance")
+            }
+            CheckBox {
+                name: "leverage"; label: qsTr("Leverages")
+            }
+        }
+
+        Group {
+            title: qsTr("Multicollinearity")
+            CheckBox { name: "tolerance"; label: qsTr("Tolerance") }
+            CheckBox { name: "vif"; label: qsTr("VIF") }
+        }
+    }
+
+    Section
+    {
+        title:		qsTr("Estimated Marginal Means and Contrast Analysis")
+        expanded:	false
```

Comment:
This is also very similar with what is inside of glmms. So if you move it into a separate file, we could then reuse it in `jaspMixedModels`.